src/main/webapp/app/course/manage/overview/course-management-card.component.html

```diff
@@ -274,6 +274,17 @@ <h4 class="text-center no-exercises mt-3 fw-medium">{{ 'artemisApp.course.noExer
             <fa-icon [icon]="faFlag"></fa-icon>
             <span [innerHTML]="'artemisApp.competency.competencyButton' | artemisTranslate" class="d-none d-xl-inline"></span>
         </a>
+        <a
+            *ngIf="course.isAtLeastInstructor"
+            [jhiFeatureToggleHide]="FeatureToggle.LearningPaths"
+            [routerLink]="['/course-management', course.id, 'learning-path-management']"
+            class="btn btn-primary me-1 mb-1 hidden-if-mobile"
+            [ngbTooltip]="'artemisApp.learningPath.learningPathButton' | artemisTranslate"
```

Comment:
is this id used anywhere?

Reply:
No, I added it to keep the buttons consistent throughout the card component.